working/augmentation-libraries/feature-specification.md

```diff
@@ -683,7 +688,9 @@ It is a **compile-time error** if:
 
 *   An augmenting variable’s initializing expression uses `augmented`, and
     the stack of augmented declarations do not include a variable with an
-    initializing expression.
+    explicit initializing expression. For nullable fields, the implicit null
```

Comment:
I think 'variables' is a better word than 'fields' here. 'Fields' is generally not used in the specification documents, presumably because 'instance variable' is a more orthogonal way to talk about them. This allows us to state rules about instance variables and static variables and library variables using just 'non-local variables' (or even just 'variables', in cases where 'non-local' can be left implicit).